spec/util/e2e-test.ts

```diff
@@ -289,7 +299,26 @@ export class IrcBridgeE2ETest {
         public readonly registration: AppServiceRegistration,
         readonly postgresDb: string,
         public readonly ircTest: TestIrcServer,
-        public readonly pool?: IrcConnectionPool) {
+        public readonly pool?: IrcConnectionPool,
+        private traceLog?: WriteStream,
+    ) {
+        const startTime = Date.now();
```

Comment:
Nitpick: this can be moved to inside the `if` block, since it's unused otherwise.

Reply:
traceLog is no longer optional, so I removed the `if`.